crates/matrix-sdk-base/src/client.rs

```diff
@@ -87,7 +87,9 @@ pub struct BaseClient {
     olm: Arc<Mutex<Option<OlmMachine>>>,
     #[cfg(feature = "encryption")]
     cryptostore: Arc<Mutex<Option<Box<dyn CryptoStore>>>>,
+    #[cfg(feature = "encryption")]
```

Comment:
Is this right? The `store_path` is used for the sled state store as well, which can be enabled without having `encryption` enabled.

Reply:
I thought I checked all usages. I'll have another look.

Reply:
I checked: With the `sled_state_store` feature, only `config.store_path` is used (which I did not touch), not `client.store_path`.

Reply:
Oh sorry, I seem to have confused those two.